modules/default/weather/providers/openweathermap.js

```diff
@@ -282,7 +282,8 @@ WeatherProvider.register("openweathermap", {
 				}
 			}
 
-			weather.precipitation = weather.rain + weather.snow;
+			weather.precipitationAmount = weather.rain + weather.snow;
+			weather.precipitationProbability = weather.pop * 100;
```

Comment:
weather.pop is never defined, so this is always empty...

Reply:
Oops, I was supposed to target the `forecast` ([documentation](https://openweathermap.org/forecast16)).<br/>
<img width="531" alt="Screenshot of documentation" src="https://user-images.githubusercontent.com/34011212/216127487-8c834249-6eaa-4be5-bc29-b68490a7fbf2.png">